demo/admin/src/dam/getRandomUnsplashImage.ts

```diff
@@ -3,6 +3,14 @@ export interface UnsplashImage {
     url: string;
 }
 
+class ExtendedFile extends File {
```

Comment:
I don't like the extending of a Browser type just to have the import source in the return value. It would be better if the package exported a type that the sample integration could use.

Reply:
0e5b5fa7586bbf623be3af508d2eb9698ff7e176